style.css

```diff
@@ -1,3 +1,3 @@
-.intro{
-    color: blue;
-}
\ No newline at end of file
+.intro {
+  color: blue;
+}
```

Comment:
- Well done on using the Microverse template for your README. However, there are still a few more things to add. Kindly ensure that your overall README is descriptive and personalized. Eg the social media handles currently have broken links and the summary 0f your project is also missing in the "About the section" of the README

Reply:
Changes made and links are updated